doc/source/cluster/kubernetes/user-guides/pod-command.md

```diff
@@ -1,18 +1,69 @@
 (kuberay-pod-command)=
 
 # Specify container commands for Ray head/worker Pods
-You can execute commands on the head/worker pods at two timings:
 
-* (1) **Before `ray start`**: As an example, you can set up some environment variables that will be used by `ray start`.
+KubeRay generates a `ray start` command for each Ray Pod.
+Sometimes, you may want to execute certain commands either before or after the ray start command, or you may wish to define the container's command yourself.
+This document shows you how to do that.
 
-* (2) **After `ray start` (RayCluster is ready)**: As an example, you can launch a Ray serve deployment when the RayCluster is ready.
+## Part 1: Specify your custom container command without any generated `ray start` command
```

Comment:
```suggestion
## Part 1: Specify a custom container command, optionally including the generated `ray start` command
```
The original title is a bit confusing because in practice, the custom container command will usually have the generated `ray start` command in it, so it's not "without" the generated command